.github/workflows/deploy-penny.yml

```diff
@@ -33,20 +38,23 @@ jobs:
           # be deployed to ECS.
           docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG -f Dockerfile .
           docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
-          echo "image=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG" >>"${GITHUB_OUTPUT}"
+          echo "image=$ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG" >> $GITHUB_OUTPUT
```

Comment:
Why?

Reply:
Probably my formatter? Not exactly sure

Reply:
Quoting the variable protects from spaces in the variable value. Adding the braces is syntactically purely stylistic in this context, but I prefer it just the same (it's "quoting" for variable names, so you could disambiguate, for example, `"$GITHUB_OUTPUT_FILE"` from `"${GITHUB_OUTPUT}_FILE"` - contrived but you get the idea).

Reply:
hmm i use it this way everywhere apparently so i guess i'll keep it this way.